requirements.yml

```
@@ -1,13 +1,13 @@
 ---
 
 - src: ome.postgresql
-  version: 4.0.0
+  version: 5.0.1
 
 - src: ome.omero_server
-  version: 3.0.0
+  version: 3.2.0
```

```suggestion
  version: 3.3.0
```
As soon as Ansible galaxy updates...